### ISSUE TITLE
Init script failure

### ISSUE THREAD

**Issue by airbornelamb** [garrows/noleg-stack#7]
After running the init script I get the below message:

It seems to have to do with this line in particular

**nginx: [emerg] BIO_new_file("/etc/nginx/lambapp.space.chained.pem") failed**

$ systemctl status nginx.service
● nginx.service - A high performance web server and a reverse proxy server
   Loaded: loaded (/lib/systemd/system/nginx.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2017-04-20 14:17:41 UTC; 18s ago
  Process: 9713 ExecStop=/sbin/start-stop-daemon --quiet --stop --retry QUIT/5 --pidfile /run/nginx.pid (code=exite
  Process: 9718 ExecStartPre=/usr/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=1/FAILURE)
 Main PID: 8410 (code=exited, status=0/SUCCESS)

Apr 20 14:17:41 lambapp systemd[1]: Starting A high performance web server and a reverse proxy server...
Apr 20 14:17:41 lambapp nginx[9718]: nginx: [emerg] BIO_new_file("/etc/nginx/lambapp.space.chained.pem") failed (SS
Apr 20 14:17:41 lambapp nginx[9718]: nginx: configuration file /etc/nginx/nginx.conf test failed
Apr 20 14:17:41 lambapp systemd[1]: nginx.service: Control process exited, code=exited status=1
Apr 20 14:17:41 lambapp systemd[1]: Failed to start A high performance web server and a reverse proxy server.
Apr 20 14:17:41 lambapp systemd[1]: nginx.service: Unit entered failed state.
Apr 20 14:17:41 lambapp systemd[1]: nginx.service: Failed with result 'exit-code'.

**Comment by garrows** [comment]
Does that file exist?

**Comment by airbornelamb** [comment]
Nope. I don't understand if it has something to do with letsencrypt or
nginx..

**Comment by garrows** [comment]
Looks like its a bug on my end. 

Remove the lines from your /etc/nginx/sites-available/lambapp.space.conf

```
    ssl_certificate     %DOMAIN%.chained.pem;
    ssl_certificate_key %DOMAIN%.private.pem;
```

Unless of course you've created your letsencrypt certs already where you should change them to point to wherever you have put your certificates. I had assumed they were put in `/etc/nginx/%DOMAIN%.chained.pem` and `/etc/nginx/%DOMAIN%.private.pem`

**Comment by airbornelamb** [comment]
Oh it may not be a bug in your end then. I had only run the init script
alone. Didn't set up anything else. Maybe I should read your readme fully
again and determine if I need to smarten up on some things.

**Comment by garrows** [comment]
No its my fault. I was halfway through adding support for letsencrypt but didn't finish. I didnt think anyone else was using this though. I'll be rebuilding some webservers in about 2 weeks so I'll have it all working then.

**Comment by airbornelamb** [comment]
Ok cool. Yeah I just discovered it and think it's awesome. Even the mean
stack doesn't really do everything. Your script is more comprehensive.
